source/host.c

```diff
@@ -953,6 +973,9 @@ void Host_Init (quakeparms_t *parms)
 	host_hunklevel = Hunk_LowMark ();
 
 	host_initialized = true;
+#ifdef __WII__
```

Comment:
Is this to hide the console? What do other platforms do?

Reply:
`VIDEO_SetBlack(false)` is a required during the initialization of an application on this platform. It is most convenient when placed here for both debugging and visual purposes. If I need to see what the console is doing during startup I can simply remove this call and observe what's happening.